sporkfish/searcher/move_ordering/composite_heuristic.py

```diff
@@ -0,0 +1,46 @@
+from enum import IntEnum
+from typing import List
+
+import chess
+
+from ...board.board import Board
+from .killer_move_heuristic import KillerMoveHeuristic
+from .mvv_lva_heuristic import MvvLvaHeuristic
+
+
```

Comment:
Nice comment tyty

You are Steve Jobs style programmer - seems you like [closed systems](https://www.quora.com/Why-did-Steve-jobs-want-a-closed-system#:~:text=Originally%20Answered%3A%20Why%20did%20Steve,you%20wanted%20it%20to%20do.)

I personally like making everything configurable with default parameter - so noob users would even know you can change certain settings - but at least open to pro user to test/play around configs. Personal preference. 

Regardless, this is something for a later PR.

Reply:
OK, I'll concede - I think this is similar to having max_depth configurable. As long as its positive, the user can pick and choose their weights and get different behaviour. Most of it should still be working. Happy to change later.

Reply:
#120.

Reply:
I see your point - think it really is a personal preference thing, and we are not really at a stage to do serious perf analysis (fine tuning parameter) - just me trash talking.

No point doing extra work with no immediate needs.